packages/mux-player/src/video-api.ts

```diff
@@ -30,6 +30,7 @@ const AllowedVideoAttributes = {
 const CustomVideoAttributes = {
   VOLUME: 'volume',
   PLAYBACKRATE: 'playbackrate',
+  PLACEHOLDER: 'placeholder',
```

Comment:
nitpick(non-blocking): These changes (`observedAttributes` & getter/setter) belong in `MuxPlayerElement`, not `VideoAPIElement`. See `PlayerAttributes` for equivalent object map.